src/convertToObject.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const finalStyle = {};
+
+  sourceString.split(';').forEach((part) => {
+    if (part.trim()) {
+      const [command, value] = part.split(':');
+
+      finalStyle[command.trim()] = value.trim();
+    }
+  });
```

Comment:
The current implementation is correct, but it can be made more readable by chaining the string methods with proper indentation, as suggested in the checklist. This can improve the readability of the code. Here's a refactored version following the checklist's good example:

const styleEntries = sourceString
  .split(';')
  .map(part => part.trim())
  .filter(part => part.length)
  .map(part => part.split(':').map(str => str.trim()));

styleEntries.forEach(([property, value]) => {
  finalStyle[property] = value;
});

This separates the concerns of splitting, trimming, and filtering the string parts from the concern of populating the 'finalStyle' object.